### PR TITLE
zh, en: update error-handling.md

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

- en: update error-handling.md
- zh: remove reasons for `DM 的全量数据导入处理模块不支持对 dump 处理模块导出的 SQL 文件进行切分。`

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/111
- Other reference link(s):

### CODE REVIEW

File: en/error-handling.md [hunk 1]
```diff
@@ -165,3 +177,29 @@ For binlog replication processing units, manually recover replication using the
 For database related passwords in all the DM configuration files, use the passwords encrypted by `dmctl`. If a database password is empty, it is unnecessary to encrypt it. For how to encrypt the plaintext password, see [Encrypt the upstream MySQL user password using dmctl](deploy-a-dm-cluster-using-ansible.md#encrypt-the-upstream-mysql-user-password-using-dmctl).
 
 In addition, the user of the upstream and downstream databases must have the corresponding read and write privileges. Data Migration also [prechecks the corresponding privileges automatically](precheck.md) while starting the data replication task.
+
+### The load processing unit reports the error `packet for query is too large. Try adjusting the 'max_allowed_packet' variable`
```
Comment (TomShawn) [inline en/error-handling.md:181]:
```suggestion
### The `load` processing unit reports the error `packet for query is too large. Try adjusting the 'max_allowed_packet' variable`
```

File: en/error-handling.md [hunk 2]
```diff
@@ -165,3 +177,29 @@ For binlog replication processing units, manually recover replication using the
...
+
+#### Reasons
+
+* Both MySQL client and MySQL/TiDB Server have the quota limit for `max_allowed_packet`. If any `max_allowed_packet` is outside the normal range, the client receives the error message. Currently, for the latest version of DM and TiDB Server, the default value of `max_allowed_packet` is `64M`.
```
Comment (TomShawn) [inline en/error-handling.md:185]:
```suggestion
* Both MySQL client and MySQL/TiDB server have the quota limits for `max_allowed_packet`. If any `max_allowed_packet` is out of either limit, the client receives the error message. Currently, for the latest version of DM and TiDB server, the default value of `max_allowed_packet` is `64M`.
```

File: en/error-handling.md [hunk 3]
```diff
@@ -165,3 +177,29 @@ For binlog replication processing units, manually recover replication using the
...
+
+* The full data import processing unit in DM does not support splitting the SQL file exported by the Dump processing unit in DM.
```
Comment (TomShawn) [inline en/error-handling.md:187]: The following content is missing in your translation:

因为 DM 的 dump 处理单元采用了最简单的编码实现，如果在 DM 实现文件切分，需要在 `TiDB Parser` 基础上实现一个完备的解析器才能正确的处理数据切分。但是随之会带来以下的问题：

    * 工作量大

    * 复杂度高，不容易保证正确性

    * 性能的极大降低

File: en/error-handling.md [hunk 4]
```diff
@@ -165,3 +177,29 @@ For binlog replication processing units, manually recover replication using the
...
+
+#### Solutions
+
+* It is recommended to set the `statement-size` option of `extra-args` for the Dump processing unit:
+
+    According to the default `--statement-size` setting, the default size of `Insert Statement` generated by the Dump processing unit is about `1M`. With this default setting, the Load processing unit does not report the error `packet for query is too large. Try adjusting the 'max_allowed_packet' variable` in most cases.
```
Comment (TomShawn) [inline en/error-handling.md:193]:
```suggestion
    According to the default `--statement-size` setting, the default size of `Insert Statement` generated by the Dump processing unit is about `1M`. With this default setting, the load processing unit does not report the error `packet for query is too large. Try adjusting the 'max_allowed_packet' variable` in most cases.
```

File: en/error-handling.md [hunk 5]
```diff
@@ -165,3 +177,29 @@ For binlog replication processing units, manually recover replication using the
...
+
+    Sometimes you might receive the following `WARN` log during the data dump. This `WARN` log does not affect the dump process. This only means that wide tables are dumped.
+
+    ```
+    Row bigger than statement_size for xxx
+    ```
+
+* If the single row of the wide table exceeds `64M`, you need to modify the following configurations and make sure the configurations take effect.
+
+    * Execute `set @@global.max_allowed_packet=134217728` (`134217728 = 128M`) in the TiDB Server.
```
Comment (TomShawn) [inline en/error-handling.md:203]:
```suggestion
    * Execute `set @@global.max_allowed_packet=134217728` (`134217728` = 128 MB) in the TiDB server.
```

File: en/error-handling.md [hunk 6]
```diff
@@ -165,3 +177,29 @@ For binlog replication processing units, manually recover replication using the
...
+
+    * First add the `max-allowed-packet: 134217728` (128M) configuration to `target-database` in the DM task configuration file. Next, execute the `stop-task` command and then execute the `start-task` command.
```
Comment (TomShawn) [inline en/error-handling.md:205]:
```suggestion
    * First add the `max-allowed-packet: 134217728` (128 MB) to the `target-database` section in the DM task configuration file. Then, execute the `stop-task` command and execute the `start-task` command.
```

File: en/error-handling.md (hunk 3 above)
Comment (toutdesuite) [reply en/error-handling.md:187]: Confirmed with the author. These can be ignored~

Comment (TomShawn) [reply en/error-handling.md:187]: Please delete this part of content as confirmed with @WangXiangUSTC

**Comment by toutdesuite** [comment]
@WangXiangUSTC PTAL, thanks~

**Pull request merged**